### PR TITLE
Update dynalite.markdown

### PR DESCRIPTION
**Pull request by ziv1234**
## Proposed change

Added the parameter level to a preset to tell the level to set the channels
when it is selected

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37533
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by ziv1234** [comment]
Why is it telling me to switch to the current?

**Comment by klaasnicolaas** [comment]
Because you didn't place a link to the core repository when creating the pull request 😉

File: source/_integrations/dynalite.markdown [hunk 1]
```diff
@@ -129,6 +129,11 @@ area:
                   required: false
                   type: float
                   default: 2.0
+                level:
+                  description: Level of the channels when the preset is selected, between 0 and 1.
+                  required: false
+                  type: float
+                  default: None
```
Comment (klaasnicolaas) [inline source/_integrations/dynalite.markdown]: If there is no default we can omit it

File: source/_integrations/dynalite.markdown [hunk 2]
```diff
@@ -179,6 +184,11 @@ preset:
           required: false
           type: float
           default: 2.0
+        level:
+          description: Level of the channels when the preset is selected, between 0 and 1.
+          required: false
+          type: float
+          default: None
```
Comment (klaasnicolaas) [inline source/_integrations/dynalite.markdown]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by ziv1234** [comment]
> Because you didn't place a link to the core repository when creating the pull request 😉

thanks :)

File: source/_integrations/dynalite.markdown (hunk 1 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: ok. will remove. I wanted to be explicit that there is no level set unless explicitly set for a preset, but agree it is confusing

File: source/_integrations/dynalite.markdown (hunk 2 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done

File: source/_integrations/dynalite.markdown (hunk 1 above)
Comment (ziv1234) [reply source/_integrations/dynalite.markdown]: done

**Review by frenck — APPROVED** [review]
Thanks, @ziv1234 👍

**Comment by ziv1234** [comment]
thanks

**Pull request merged**